cli/cmd/component.go

```diff
@@ -359,21 +369,49 @@ func runComponentsInstall(_ *cobra.Command, args []string) (err error) {
 		return
 	}
 
-	component, found := cli.LwComponents.GetComponent(args[0])
+	component_name = args[0]
+
+	component, found := cli.LwComponents.GetComponent(component_name)
 	if !found {
-		err = errors.New("component not found. Try running 'lacework component list'")
+		err = errors.New(fmt.Sprintf("component %s not found. Try running 'lacework component list'", component_name))
 		return
 	}
 
+	cli.OutputChecklist(successIcon, fmt.Sprintf("Component %s found\n", component_name))
+
+	if versionArg == "" {
+		version = component.LatestVersion.String()
+	} else {
+		version = versionArg
```

Comment:
What do we gain having this if statement all the way here? Should we instead initialize version with the context of versionArg and then, if version is empty then get the latest version? Another nit but this if confused me a bit.